recipes/pybufr-ecmwf/meta.yaml

```diff
@@ -3,46 +3,40 @@
 {% set sha2563k = "b4027e018188a1ea29ab3fb5374087bac569c290335a1d5cde8356b446069435" %}
 
 package:
-  name: pybufr-ecmwf  # [py2k]
-  name: pybufr-ecmwf3  # [py3k]
+  name: pybufr-ecmwf
   version: {{ version }}
 
 source:
-  fn: pybufr-ecmwf-{{ version }}.tar.gz  # [py2k]
-  fn: pybufr-ecmwf3-{{ version }}.tar.gz  # [py3k]
+  fn: pybufr-ecmwf-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pybufr-ecmwf/pybufr-ecmwf-{{ version }}.tar.gz  # [py2k]
   url: https://pypi.io/packages/source/p/pybufr-ecmwf3/pybufr-ecmwf3-{{ version }}.tar.gz  # [py3k]
   sha256: {{ sha256 }}  # [py2k]
   sha256: {{ sha2563k }}  # [py3k]
 
 build:
-  skip: true  # [win]
+  skip: True  # [win]
   number: 0
-  script:
-    - python clean.py
-    - python setup.py install
+  script: python setup.py install
 
 requirements:
   build:
-    - numpy x.x
     - python
+    - numpy x.x
     - gcc
 
   run:
-    - numpy x.x
     - python
-    - libgfortran 
+    - numpy x.x
+    - libgfortran
```

Comment:
The `import` test worked. But we are still unclear if `pybufr-ecmwf` needs `libquamath.so` (and hence the full `libgcc` instead of only `libgfortran` here.) Maybe we could come up with more tests?